database/insertData.js

```diff
@@ -12,6 +12,7 @@ const queries = [
         template: {
           code: "UserRego1"
           name: "User Registration"
+          isLinear: true 
```

Comment:
If you set the default as suggested above, then you can omit this line.

Reply:
Done! Thanks